SimplenoteUITests/SimplenoteUISmokeTestsSettings.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+class SimplenoteUISmokeTestsSettings: XCTestCase {
+
+    override class func setUp() {
+        app.launch()
+        getToAllNotes()
+        _ = attemptLogOut()
+        EmailLogin.open()
+        EmailLogin.logIn(email: testDataExistingEmail, password: testDataExistingPassword)
+        NoteList.waitForLoad()
+    }
+
+    override func setUpWithError() throws {
+        getToAllNotes()
+        NoteList.trashAllNotes()
+        Trash.empty()
+        NoteList.openAllNotes()
+    }
+
+    override class func tearDown() {
+        Settings.open()
+        Settings.condensedModeDisable()
+        Settings.close()
+    }
+
+    func testUsingCondensedNoteList() throws {
+        trackTest()
+        let note = NoteData(
+            name: "Condensed Mode Test",
+            content: "Condensed Mode Content",
+            tags: []
+        )
```

Comment:
This makes the tests much tidier 🙇

Reply:
Glad to see it's useful already 😄 

We could make it even tidier by using a default value for the tags.

```swift
struct NoteData {

   let ...

   init(name: String, content: String, tags: [String] = []) {
      ...
```

This way, we'd only have to add tags when they matter and this code would become
```suggestion
        let note = NoteData(
            name: "Condensed Mode Test",
            content: "Condensed Mode Content"
        )
```

Reply:
Thank you! I just learned that I could also define the default value for tag using by the variable instead of constant, and providing the default value:

`var tags: [String] = []`

In that case it's not needed to initialize all constants, as with `init` approach. Are there any trade-offs I did not consider here (for this case)?